allauth/account/templatetags/account_tags.py

```diff
@@ -3,4 +3,4 @@
 warnings.warn("{% load account_tags %} is deprecated, use {% load account %}",
               DeprecationWarning)
 
-from account import *  # noqa
+from .account import *  # noqa
```

Comment:
Ah, in that case a PR is probably required here: https://github.com/pydanny/cookiecutter-django/blob/master/%7B%7Bcookiecutter.repo_name%7D%7D/%7B%7Bcookiecutter.repo_name%7D%7D/templates/account/email.html#L5